### DISCUSSION TITLE
Storybook Usage

### DISCUSSION THREAD

**Discussion by daarond** [vuejs/pinia#523] [Help and Questions]
I'm trying to show a component in Storybook (https://storybook.js.org/), but the store is not getting set up. At the import line (import { defineStore } from 'pinia') I get the error "Invalid value used as weak map key."

Any suggestions?

**Comment by posva** [comment]
You need to install the pinia plugin before creating a store.
Unfortunately I don't know storybook enough to tell where this has to be done
